src/azul/service/manifest_controller.py

```diff
@@ -167,11 +164,6 @@ def get_manifest_async(self,
             return Response(body=body)
         else:
             headers = {k: str(body[k]) for k in body.keys() & {'Location', 'Retry-After'}}
-            return Response(body=''.join(
-                dedent(
-                    f'''Download the manifest in {shell} with `curl` using:
-                        {cmd}
-                    '''
-                ) for shell, cmd in command_lines.items()),
-                status_code=body['Status'],
-                headers=headers)
+            msg = '\n'.join(f'Download the manifest in {shell} with `curl` using:\n{cmd}'
```

Comment:
W

https://github.com/DataBiosphere/azul/blob/develop/CONTRIBUTING.rst#14line-wrapping-and-indentation

Reply:
This is a generator expression.